### DISCUSSION TITLE
UI Thoughts

### DISCUSSION THREAD

**Discussion by zxcaf** [afadil/wealthfolio#103] [Ideas]
Nicety: From the **Holdings > Overview**
- it would be nice for the individual holdings in the **Holdings** section at the bottom to link the individual holding details

Consistency: From the **Holdings > Holdings**
- Each individual holding, the **>** (on the far right) takes you to that individual holding's details, however the **<-** (top left) button when in the individual details screen, takes you back to the **Holdings > Overview**, rather than where you just came from **Holdings/Holdings**

Nicety: More General UI Thoughts
- On the Dashboard Screen, and in the details of an individual holding's details, it would be nice to have a "1D" (one day) option, and perhaps a "YTD" (year to date) option.

**Comment by afadil** [comment]
- First two points are now included in v1.0.18
